[E 250518 00:43:13 smartConnect:222] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/secure/angelbroking/order/v1/placeOrder. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'e4:c7:67:68:e7:86', 'Accept': 'application/json', 'X-PrivateKey': 'BPPQBmHy', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlY0MDgwOSIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPakV3TWl3aWMyOTFjbU5sSWpvaU15SXNJbVJsZG1salpWOXBaQ0k2SWpVd09URXpZMkV3TFRZd016Z3RNemcwTUMxaU9XRXhMVGMwTlRCaFlqY3haak0wWkNJc0ltdHBaQ0k2SW5SeVlXUmxYMnRsZVY5Mk1pSXNJbTl0Ym1WdFlXNWhaMlZ5YVdRaU9qRXdNaXdpY0hKdlpIVmpkSE1pT25zaVpHVnRZWFFpT25zaWMzUmhkSFZ6SWpvaVlXTjBhWFpsSW4wc0ltMW1JanA3SW5OMFlYUjFjeUk2SW1GamRHbDJaU0o5ZlN3aWFYTnpJam9pZEhKaFpHVmZiRzluYVc1ZmMyVnlkbWxqWlNJc0luTjFZaUk2SWxZME1EZ3dPU0lzSW1WNGNDSTZNVGMwTnpVNE56WXhNeXdpYm1KbUlqb3hOelEzTlRBeE1ETXpMQ0pwWVhRaU9qRTNORGMxTURFd016TXNJbXAwYVNJNklqTTVNbUl5TVdRd0xXVTRNR010TkRkaFpDMDVZbUl4TFRrMk1HWmtPV1ExWlRZd1pDSXNJbFJ2YTJWdUlqb2lJbjAuTHZQaWxjWThyRER3MUR1Uy03c0pySm9HWXlzbDlHaEppRWU5Y0h4SDdWcWMxMHdaZ1lRTlhxbjlNU2tybDhfMnBFZTJoSEdBR3A1dmQ1UWs4cTRZZTZhUkc4aENWbnRnOUxPQVNXQkJja2dVb3NQekZSX2JMTkVBYlpsRjZBQTg3dzNsRUh3WTR3cXd1REo3dlZGaW81aGI3TG9sYzVnMUtwRk1pamd2VWJZIiwiQVBJLUtFWSI6IkJQUFFCbUh5IiwiaWF0IjoxNzQ3NTAxMjEzLCJleHAiOjE3NDc1ODc2MTN9.8MU28i762DctULBXOiXFijag_AF3VAdQ17v1tEt1_kUEfgTqBVPYnwYfEkRPwikO8I7VMJ-dxFCtoL5vKkxmpg'}, Request: {'variety': 'NORMAL', 'tradingsymbol': 'NIFTY29MAY25FUT', 'symboltoken': '57133', 'transactiontype': 'BUY', 'exchange': 'NFO', 'ordertype': 'MARKET', 'producttype': 'INTRADAY', 'duration': 'DAY', 'price': 0.0, 'squareoff': '0', 'stoploss': '0', 'quantity': 75}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/order/v1/placeOrder (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x000001ED4E7751D0>, 'Connection to apiconnect.angelone.in timed out. (connect timeout=7)'))
[E 250518 00:47:27 smartConnect:222] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/secure/angelbroking/order/v1/placeOrder. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'e4:c7:67:68:e7:86', 'Accept': 'application/json', 'X-PrivateKey': 'BPPQBmHy', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IlY0MDgwOSIsInJvbGVzIjowLCJ1c2VydHlwZSI6IlVTRVIiLCJ0b2tlbiI6ImV5SmhiR2NpT2lKU1V6STFOaUlzSW5SNWNDSTZJa3BYVkNKOS5leUoxYzJWeVgzUjVjR1VpT2lKamJHbGxiblFpTENKMGIydGxibDkwZVhCbElqb2lkSEpoWkdWZllXTmpaWE56WDNSdmEyVnVJaXdpWjIxZmFXUWlPakV3TWl3aWMyOTFjbU5sSWpvaU15SXNJbVJsZG1salpWOXBaQ0k2SWpVd09URXpZMkV3TFRZd016Z3RNemcwTUMxaU9XRXhMVGMwTlRCaFlqY3haak0wWkNJc0ltdHBaQ0k2SW5SeVlXUmxYMnRsZVY5Mk1pSXNJbTl0Ym1WdFlXNWhaMlZ5YVdRaU9qRXdNaXdpY0hKdlpIVmpkSE1pT25zaVpHVnRZWFFpT25zaWMzUmhkSFZ6SWpvaVlXTjBhWFpsSW4wc0ltMW1JanA3SW5OMFlYUjFjeUk2SW1GamRHbDJaU0o5ZlN3aWFYTnpJam9pZEhKaFpHVmZiRzluYVc1ZmMyVnlkbWxqWlNJc0luTjFZaUk2SWxZME1EZ3dPU0lzSW1WNGNDSTZNVGMwTnpVNE56WXhNeXdpYm1KbUlqb3hOelEzTlRBeE1ETXpMQ0pwWVhRaU9qRTNORGMxTURFd016TXNJbXAwYVNJNklqTTVNbUl5TVdRd0xXVTRNR010TkRkaFpDMDVZbUl4TFRrMk1HWmtPV1ExWlRZd1pDSXNJbFJ2YTJWdUlqb2lJbjAuTHZQaWxjWThyRER3MUR1Uy03c0pySm9HWXlzbDlHaEppRWU5Y0h4SDdWcWMxMHdaZ1lRTlhxbjlNU2tybDhfMnBFZTJoSEdBR3A1dmQ1UWs4cTRZZTZhUkc4aENWbnRnOUxPQVNXQkJja2dVb3NQekZSX2JMTkVBYlpsRjZBQTg3dzNsRUh3WTR3cXd1REo3dlZGaW81aGI3TG9sYzVnMUtwRk1pamd2VWJZIiwiQVBJLUtFWSI6IkJQUFFCbUh5IiwiaWF0IjoxNzQ3NTAxMjEzLCJleHAiOjE3NDc1ODc2MTN9.8MU28i762DctULBXOiXFijag_AF3VAdQ17v1tEt1_kUEfgTqBVPYnwYfEkRPwikO8I7VMJ-dxFCtoL5vKkxmpg'}, Request: {'variety': 'NORMAL', 'tradingsymbol': 'NIFTY29MAY25FUT', 'symboltoken': '57133', 'transactiontype': 'BUY', 'exchange': 'NFO', 'ordertype': 'MARKET', 'producttype': 'CARRYFORWARD', 'duration': 'DAY', 'price': 0.0, 'squareoff': '0', 'stoploss': '0', 'quantity': 75}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/order/v1/placeOrder (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x000001ED4E53BC50>, 'Connection to apiconnect.angelone.in timed out. (connect timeout=7)'))
[E 250518 01:07:14 smartConnect:523] Error occurred in ind_order_details: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/order/v1/details/0518bc9f241bAO (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x000002CF77F0D090>, 'Connection to apiconnect.angelone.in timed out. (connect timeout=None)'))
[E 250518 12:56:51 smartWebSocketV2:212] Error occurred during subscribe: 'NoneType' object has no attribute 'extend'
[W 250518 12:56:51 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[E 250518 12:57:02 smartWebSocketV2:212] Error occurred during subscribe: 'NoneType' object has no attribute 'extend'
[W 250518 12:57:02 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[E 250518 12:57:12 smartWebSocketV2:212] Error occurred during subscribe: 'NoneType' object has no attribute 'extend'
[W 250518 12:57:12 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 12:57:12 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 12:57:12 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 12:57:12 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 12:57:12 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 12:57:12 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 12:57:12 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[E 250518 12:58:44 smartWebSocketV2:212] Error occurred during subscribe: 'NoneType' object has no attribute 'extend'
[W 250518 12:58:44 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 12:58:52 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 12:58:52 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[E 250518 13:00:33 smartWebSocketV2:212] Error occurred during subscribe: 'NoneType' object has no attribute 'extend'
[W 250518 13:00:33 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[E 250518 13:00:43 smartWebSocketV2:212] Error occurred during subscribe: 'NoneType' object has no attribute 'extend'
[W 250518 13:00:43 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 13:00:46 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 13:00:46 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 13:00:46 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 13:00:46 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[E 250518 13:00:49 smartWebSocketV2:212] Error occurred during subscribe: 'NoneType' object has no attribute 'extend'
[W 250518 13:00:49 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 13:00:56 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 13:00:56 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[E 250518 13:01:40 smartWebSocketV2:212] Error occurred during subscribe: 'NoneType' object has no attribute 'extend'
[W 250518 13:01:40 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 13:01:44 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 13:01:44 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[E 250518 13:04:07 smartWebSocketV2:212] Error occurred during subscribe: 'NoneType' object has no attribute 'extend'
[W 250518 13:04:07 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 13:04:08 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 13:04:10 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[E 250518 13:04:38 smartWebSocketV2:212] Error occurred during subscribe: 'NoneType' object has no attribute 'extend'
[W 250518 13:04:38 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 13:04:40 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 13:04:40 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[E 250518 13:09:45 smartWebSocketV2:212] Error occurred during subscribe: 'NoneType' object has no attribute 'extend'
[W 250518 13:09:45 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 13:09:50 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 13:09:50 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[E 250518 13:12:18 smartWebSocketV2:212] Error occurred during subscribe: 'NoneType' object is not iterable
[W 250518 13:12:18 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 13:12:21 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 13:12:21 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[E 250518 13:20:08 smartWebSocketV2:212] Error occurred during subscribe: 'NoneType' object is not iterable
[W 250518 13:20:08 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 13:20:11 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 13:20:11 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[E 250518 13:23:47 smartWebSocketV2:212] Error occurred during subscribe: 'NoneType' object is not iterable
[W 250518 13:23:47 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 13:23:50 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 13:23:51 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[E 250518 13:24:26 smartWebSocketV2:212] Error occurred during subscribe: 'NoneType' object is not iterable
[W 250518 13:24:26 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[E 250518 13:24:36 smartWebSocketV2:212] Error occurred during subscribe: 'NoneType' object is not iterable
[W 250518 13:24:36 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 13:24:39 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 13:24:39 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 13:24:39 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[E 250518 13:41:12 smartWebSocketV2:212] Error occurred during subscribe: 'NoneType' object is not iterable
[W 250518 13:41:12 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 13:41:14 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 13:41:15 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 13:46:38 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 13:46:38 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[E 250518 13:47:50 smartWebSocketV2:212] Error occurred during subscribe: 'NoneType' object is not iterable
[W 250518 13:47:50 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 13:47:54 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 13:47:54 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[E 250518 13:49:24 smartWebSocketV2:212] Error occurred during subscribe: 'NoneType' object is not iterable
[W 250518 13:49:24 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 13:49:27 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 13:49:27 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[E 250518 13:49:38 smartWebSocketV2:212] Error occurred during subscribe: 'NoneType' object is not iterable
[W 250518 13:49:38 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 13:49:40 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 13:49:40 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 13:50:58 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 13:50:58 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 14:49:18 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 14:49:18 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 14:50:06 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 14:50:07 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 14:56:04 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 14:56:08 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 14:56:08 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 14:56:41 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 14:56:52 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 14:57:02 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 14:57:02 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 14:57:02 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 14:57:02 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 14:57:36 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 14:57:37 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 15:00:39 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 15:01:10 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 15:01:10 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 15:01:10 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 15:01:34 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 15:01:34 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 15:02:47 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 15:02:47 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 16:46:27 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 17:05:26 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 17:17:47 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 17:17:47 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 17:17:47 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 17:17:47 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 17:20:01 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 17:20:08 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 17:20:21 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 17:21:21 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 17:21:21 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 17:21:21 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 17:27:36 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 17:27:47 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 17:27:56 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 17:27:56 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 17:27:56 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 17:29:30 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 17:29:30 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 17:30:07 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 17:30:15 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 17:30:18 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 17:30:23 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 17:30:23 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 17:33:14 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 17:33:15 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 17:37:07 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 17:37:08 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 17:41:20 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 17:41:20 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 17:54:04 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 1)...
[W 250518 17:54:14 smartWebSocketV2:319] Attempting to resubscribe/reconnect (Attempt 2)...
[W 250518 17:54:24 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 17:54:24 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 17:54:24 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[W 250518 17:54:24 smartWebSocketV2:343] Connection closed due to max retry attempts reached.
[E 250518 18:34:11 smartConnect:247] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/auth/angelbroking/user/v1/loginByPassword. Error: Invalid totp. URL: https://apiconnect.angelone.in/rest/auth/angelbroking/user/v1/loginByPassword, Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'e4:c7:67:68:e7:86', 'Accept': 'application/json', 'X-PrivateKey': '1234', 'X-UserType': 'USER', 'X-SourceID': 'WEB'}, Request: {'clientcode': 'Hunter', 'password': 'Hunter@0606', 'totp': '797053'}, Response: {'status': False, 'message': 'Invalid totp', 'errorcode': 'AB1050', 'data': None}
